src/schema/contracts/common/signature.ts

```diff
@@ -18,6 +18,7 @@ import { resolveOrGenerateId } from "../../../common/signature-minting";
 export const BaseSignaturePayloadInput = z.object({
   to: z.string().default(constants.AddressZero),
   price: PriceSchema.default(0),
+  pricePerToken: PriceSchema.default(0),
```

Comment:
hmmm i don't really like this - now I have to choose between `price` or `pricePerToken` - can we not just use one (probably pricePerToken) and do the mapping ourselves?